Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalPedestal_cff.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+#------------------------------------------------
+#AlCaReco filtering for HCAL minbias:
+#------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.ALCARECOHcalCalMinBias_cff import *
+hcalDigiAlCaMB.InputLabel = 'hltHcalCalibrationRaw'
+from EventFilter.HcalRawToDigi.HcalRawToDigi_cfi import *
+#hcalDigis.InputLabel = 'hltHcalCalibrationRaw'
```

Comment:
@kodolova 
Removing the definition of `hcalDigis.InputLabel` breaks the standard sequence definition:

````
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/bin/slc6_amd64_gcc491/cmsDriver.py", line 43, in <module>
    run()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/bin/slc6_amd64_gcc491/cmsDriver.py", line 15, in run
    configBuilder.prepare()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/Applications/ConfigBuilder.py", line 2084, in prepare
    self.addStandardSequences()
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/Applications/ConfigBuilder.py", line 717, in addStandardSequences
    getattr(self,"prepare_"+stepName)(sequence = '+'.join(stepSpec))
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/Applications/ConfigBuilder.py", line 1228, in prepare_ALCA
    alcaConfig=self.loadDefaultOrSpecifiedCFF(sequence,self.ALCADefaultCFF)
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/Applications/ConfigBuilder.py", line 1181, in loadDefaultOrSpecifiedCFF
    l=self.loadAndRemember(defaultCFF,unsch)
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/Applications/ConfigBuilder.py", line 290, in loadAndRemember
    self.process.load(includeFile)
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/FWCore/ParameterSet/Config.py", line 515, in load
    self.extend(sys.modules[moduleName])
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/FWCore/ParameterSet/Config.py", line 532, in extend
    self.__setattr__(name,item)
  File "/afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/FWCore/ParameterSet/Config.py", line 364, in __setattr__
    raise ValueError(msg1+s.label_()+msg2)
ValueError: Trying to override definition of hcalDigis while it is used by the sequence RawToDigi
 new object defined in: /afs/cern.ch/cms/sw/ReleaseCandidates/vol0/slc6_amd64_gcc491/cms/cmssw-patch/CMSSW_7_5_X_2015-03-18-1400/python/EventFilter/HcalRawToDigi/HcalRawToDigi_cfi.py
 existing object defined in: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_5_X_2015-03-18-1400/python/Configuration/StandardSequences/RawToDigi_cff.py
````